laa_court_data_api_app/routers/hearing_summaries.py

```diff
@@ -32,11 +32,11 @@ async def get_hearing_summaries(urn: str):
             logging.info(f"Hearing_Summaries_To_Show: {summaries.count}")
             return HearingSummariesResponse(hearing_summaries=summaries)
         case 400:
-            logging.info("Prosecution_Case_Endpoint_Validation_Failed")
+            logging.error("Prosecution_Case_Endpoint_Validation_Failed")
```

Comment:
This is not an error. Validation failing does not constitute as the application falling over - therefor info should be the correct level here.